hbase-shell/src/main/ruby/shell/commands/move.rb

```diff
@@ -30,10 +30,13 @@ def help
 the encoded region name portion is 527db22f95c8a9e0116f0cc13c680396
 A server name is its host, port plus startcode. For example:
 host187.example.com,60020,1289493121758
+To move meta region, pass 'hbase:meta' as encoded region name"
 Examples:
 
   hbase> move 'ENCODED_REGIONNAME'
   hbase> move 'ENCODED_REGIONNAME', 'SERVER_NAME'
+  hbase> move 'hbase:meta'
```

Comment:
Good to add one comment stating that `hbase:meta` is the only tablename allowed because it has single region so far

Reply:
Done